app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= button_to( "Log Out",destroy_user_session_path, method: :delete) %> 
```

Comment:
- [ ] Kindly include the Log Out button inside the header to maintain the order with the original design for a better Ui 👇🏽 

| Original design | Your design |
|----|----|
| ![image](https://github.com/prius2055/rails-budget-app/assets/58642949/a65809e3-6162-4bfc-adcd-12fba7812f9c) | ![image](https://github.com/prius2055/rails-budget-app/assets/58642949/6949e12b-3060-40bc-94bc-a000813bea3a) |

Reply:
Fixed. Thank you